txclib/commands.py

```diff
@@ -92,18 +81,41 @@ def cmd_init(argv, path_to_tx):
     prj = project.Project(path_to_tx)
     prj.getset_host_credentials(transifex_host, username=options.user,
                                 password=options.password,
-                                token=options.token, save=save)
+                                token=options.token,
+                                no_interactive=options.no_interactive)
     prj.save()
-    logger.info("Done.")
+
+    if not options.skipsetup and not options.no_interactive:
+        logger.info(messages.running_tx_set)
+        cmd_set([], path_to_tx)
+    else:
+        logger.info("Done.")
 
 
 def cmd_set(argv, path_to_tx):
     """Add local or remote files under transifex"""
     parser = set_parser()
-    (options, args) = parser.parse_args(argv)
+    wizard_run = False
+    if len(argv) == 0:
+        # Run the wizard and configure parse with the wizard inputs
+        try:
+            wizard_options = Wizard(path_to_tx).run()
+            wizard_run = True
+        except SystemExit:
+            print("\n")
+            exit(1)
```

Comment:
What's the point of this exactly?

Reply:
<img width="746" alt="screen shot 2017-11-22 at 12 38 00" src="https://user-images.githubusercontent.com/2773012/33123179-2004eb0a-cf82-11e7-80d4-44a2018bb0ef.png">
This is so that the above don't happen when someone `Ctrl+c` while in the interactive wizard. No other reason.

Reply:
tbh, doesn't seem to justify it. but ok.

Reply:
Ctrl+C doesn't raise SystemExit. It raises KeyboardInterrupt. Also, you want `sys.exit` not `exit` here. AFAICS you changed the wizard code to reraise whatever exception it catches (instead of the `exit(1)` it was calling. Therefore you might want to catch a general `Exception` here?

Reply:
@kouk It actually raises SystemExit, probably due to this snippet here: https://github.com/transifex/transifex-client/blob/master/txclib/cmdline.py#L28.

Reply:
also there already a generic exception handling here https://github.com/transifex/transifex-client/blob/master/txclib/cmdline.py#L126, that's why I am only handling the SystemExit.

Reply:
I fixed the `sys.exit` issue though